src/services/ResponseTypes.ts

```diff
@@ -20,12 +20,26 @@ export enum StatusState {
 export type Response = {
 	message: string;
 };
+export const OrderedInputFilesNames = [
+	'info.json',
+	'geo.dat',
+	'mat.dat',
+	'beam.dat',
+	'detect.dat',
+	'sobp.dat'
```

Comment:
This is for a future issue: filename `sobp.dat` should not be hardcoded.
SHIELD-HIT12A "project" is a collection of any number of files with arbitrary names, user can specify in `beam.dat` that `USECBEAM plik.dat` and upload file named `plik.dat`. 

The only names which are "fixed" are 4 mandatory input files, rest is of any name.

Reply:
This is just for ordering and autocompletion purposes. It doesn't appear if the user won't provide it.

Reply:
If we ever need to mock input files in code to check something then IDE can suggest names that we might want to include
![image](https://user-images.githubusercontent.com/26445545/232475664-37dde86f-bcad-427d-b89f-23f7a22294c5.png)
There is no difference for the users with or without sobp.dat filename